backend/core/src/shared/entities/address.entity.ts

```diff
@@ -55,14 +55,14 @@ export class Address {
   @Expose()
   @IsOptional({ groups: [ValidationsGroupsEnum.partners] })
   @IsString({ groups: [ValidationsGroupsEnum.default] })
-  @MaxLength(64, { groups: [ValidationsGroupsEnum.default] })
+  @MaxLength(128, { groups: [ValidationsGroupsEnum.default] })
```

Comment:
Does this also need a migration?

Reply:
No, it looks like this is just a typeorm-level validation, not a DB-level validation (`yarn db:migration:generate` doesn't generate anything for this)